holmes/core/toolset_manager.py

```
@@ -111,19 +114,21 @@ def _list_all_toolsets(
+        final_toolsets = list(toolsets_by_name.values())
+        self._apply_global_transformer_configs(final_toolsets)
+
         # check_prerequisites against each enabled toolset
         if not check_prerequisites:
-            return list(toolsets_by_name.values())
-
-        enabled_toolsets: List[Toolset] = []
+            return final_toolsets
         for _, toolset in toolsets_by_name.items():
             if toolset.enabled:
                 enabled_toolsets.append(toolset)
```

Ruff (0.12.2)

Undefined name `enabled_toolsets`

(F821)